### PR TITLE
Create a labels stream

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: tap_linear/streams.py [hunk 1]
```diff
@@ -55,3 +65,30 @@ class CommentsStream(LinearStream):
...
     query = commentsQuery
+
+class LabelsStream(LinearStream):
+
+    name = "labels"
+    schema = labelsSchema
+    primary_keys = ["id"]
+    replication_key = "updatedAt"
+    query = labelsQuery
+    parent_stream_type = TeamsStream
```
Comment (hsyyid) [inline tap_linear/streams.py:76]:
```py
ignore_parent_replication_keys = True
```

File: tap_linear/streams.py [hunk 2]
```diff
@@ -47,6 +51,12 @@ class TeamsStream(LinearStream):
     replication_key = "updatedAt"
     query = teamsQuery
 
+    def get_child_context(self, record: dict, context) -> dict:
+        """Return a context dictionary for child streams."""
+        return {
+            "variables":{"teamId": record["id"]}
```
Comment (hsyyid) [inline tap_linear/streams.py]: I would make the `teamId` top level in this object

File: tap_linear/client.py [hunk 3]
```diff
@@ -58,9 +58,12 @@ def prepare_request_payload(
             "%Y-%m-%dT%H:%M:%SZ"
         )
         self.logger.info(f"Previous state timestamp: {value}")
+        variables = {"next": next_page_token, "replicationKeyValue": value}
+        if context and 'variables' in context:
+            variables.update(context['variables'])
```
Comment (hsyyid) [inline tap_linear/client.py]:
```
if context:
   variables.update(context)
```

File: tap_linear/schemas/labels.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from singer_sdk import typing as th
+
+labelsSchema = th.PropertiesList(
+    th.Property("id", th.StringType),
+    th.Property("name", th.StringType),
+    th.Property("description", th.StringType),
+    th.Property("color", th.StringType),
+    th.Property("createdAt", th.DateTimeType),
+    th.Property("updatedAt", th.DateTimeType),
+    th.Property("archivedAt", th.DateTimeType),
```
Comment (hsyyid) [inline tap_linear/schemas/labels.py:10]: Just add `teamId`, it'll be auto-injected from the context (I think)

**Review by hsyyid — CHANGES_REQUESTED** [review]

File: tap_linear/streams.py (hunk 1 above)
Comment (ghost) [reply tap_linear/streams.py:76]: Done

File: tap_linear/streams.py (hunk 2 above)
Comment (ghost) [reply tap_linear/streams.py]: Done

File: tap_linear/client.py (hunk 3 above)
Comment (ghost) [reply tap_linear/client.py]: Done

File: tap_linear/schemas/labels.py (hunk 4 above)
Comment (ghost) [reply tap_linear/schemas/labels.py:10]: Done

**Review by hsyyid — APPROVED** [review]

**Pull request merged**